### PR TITLE
Hotfix/1.3.3

### PR DESCRIPTION
**Pull request by kmcelwee**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -18,13 +19,6 @@ a `DH project sponsored by CDH <https://cdh.princeton.edu/projects/derridas-marg
    :target: https://codecov.io/gh/Princeton-CDH/derrida-django
    :alt: Code coverage
 
-.. image:: https://api.codeclimate.com/v1/badges/1cb1a007da663863e326/maintainability
-   :target: https://codeclimate.com/github/Princeton-CDH/derrida-django/maintainability
-   :alt: Maintainability
-
-.. image:: https://requires.io/github/Princeton-CDH/derrida-django/requirements.svg?branch=main
-   :target: https://requires.io/github/Princeton-CDH/derrida-django/requirements/?branch=main
-   :alt: Requirements Status
```
Comment (rlskoeser) [inline README.rst:27]: Thanks for removing these!

File: CITATION.cff [hunk 2]
```diff
@@ -30,7 +30,11 @@ authors:
   family-names: Bauer
   affiliation: Princeton University
   orcid: https://orcid.org/0000-0002-4026-740X
-version: '1.2'
+- given-names: Kevin
+  family-names: McElwee
+  affiliation: '@Princeton-CDH'
```
Comment (rlskoeser) [inline CITATION.cff:35]: Please update affiliation to "Princeton University" for everyone who still works here (I guess just you and me!?) and remove affiliation for everyone else.

**Review by rlskoeser — APPROVED** [review]
Great! Please update affiliations in the citation file before merging, everything else is good.

File: README.rst (hunk 1 above)
Comment (kmcelwee) [reply README.rst:27]: This was actually a commit you made 😄

Comment (rlskoeser) [reply README.rst:27]: 🤣

**Pull request merged**